com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/TessellationShare.hlsl

```diff
@@ -161,6 +161,7 @@ PackedVaryingsToPS Domain(TessellationFactors tessFactors, const OutputPatch<Pac
                                                     p0, p1, p2, n0, n1, n2,
                                                     baryCoords, _TessellationShapeFactor);
 #endif
+    varying.vmesh.positionPredisplacementRWS = varying.vmesh.positionRWS;
```

Comment:
don't you miss a #ifdef VARYINGS_NEED_POSITIONPREDISPLACEMENT_WS here?

Reply:
Practically speaking, I don't think it should make any actual difference, since it won't actually get propagated. I just always have positionPredisplacementRWS in the struct, rather than making it optional to write to it. That does mean it also ends up the struct in VaryingMesh.hlsl, do I'm not averse to putting a guard around it too, then it's only declared in Shader Graph.

Reply:
Now guarded.